### ISSUE TITLE
Should hugectr add  batch normalization offset and scale

### ISSUE THREAD

**Issue by BookChan** [NVIDIA-Merlin/HugeCTR#55]
Saving hugectr model with the batch normalization layer,  we can get   `gamma, beta`  but not `offset, scale `, which should be estimator of the training data:
![image](https://user-images.githubusercontent.com/7218634/98760578-88e84e80-240e-11eb-818e-93920dc637e3.png)
![image](https://user-images.githubusercontent.com/7218634/98760389-22633080-240e-11eb-8e20-bec9786c445e.png) 

And when we transfer hugectr model to tensorflow model,  we need to  set offset and scale in    `tf.nn.batch_normalization(
    x, mean, variance, offset, scale, variance_epsilon, name=None
)` .
**Can hugectr adds offset and scale parameters to be saved to the binary model?**
